app/main.py

```diff
@@ -1 +1,51 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        for skill in ["JavaScript", "HTML", "CSS"]:
+            self.learn_skill(skill)
```

Comment:
Use method `extend` of list
```suggestion
        self.skills.extend(["JavaScript", "HTML", "CSS"])
```